File: python_programs/calculadora_imc.py
peso = float(input("Digite seu peso (ex: 60.4), em kg: "))
altura = float(input("Digite sua altura (ex: 1.61), em metros: "))
imc = peso /(altura**2)
print("o seu imc é", imc)
print("o seu imc é", "%.2f" %imc) ##printf-style String Formatting
print("O seu IMC é", round(imc, 2)) #função round
print(f'O seu imc é {imc:.2f}') #F-Strings

File: C_programs/calculadora_imc.c
#include <stdio.h>

int main()
{
    float peso, altura, imc;

    printf("Digite seu peso, em Kg: ");
    scanf("%f", &peso);
    printf("Digite sua altura, em m (Ex.:1.61): ");
    scanf("%f", &altura);

    imc = peso/(altura * altura);

    printf("O seu imc eh %.2f", imc);

    return 0;
}